components/mailbox/src/Mailbox.svelte

```diff
@@ -392,6 +393,20 @@
     returnToMailbox();
   }
 
+  async function downloadSelectedFile(event: CustomEvent) {
+    const file = event.detail.file;
+    const downloadedFileData = await downloadFile({
+      file_id: file.id,
+      component_id: id,
+      access_token,
+    });
+    var a = document.createElement("a");
```

Comment:
```suggestion
    let a = document.createElement("a");
```

Reply:
@pooja169usp I don't think this trick works on iOS/Safari. Clicking it does nothing.

Reply:
I'll take a look

Reply:
@AaronDDM I am able to download the file. 
![image](https://user-images.githubusercontent.com/16315004/144621446-dac49f34-fb05-4480-8f11-176b6be126cc.png)

Is it possible that your pop up is blocked?

Reply:
I tried this on my Mobile device. AFAIK, iOS applies some additional security checks when you attempt to mimic touch/mouse clicks via JS.

Reply:
Fixed the pdf download isssue on ios safari:
![image](https://user-images.githubusercontent.com/16315004/144696266-49d128c4-cd6f-40b0-80dc-2de30bed56df.png)